src/components/Post-list/Post/Post.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { User } from './User/User';
+import { CommentsList } from './Comments-list/Commets-list';
```

Comment:
```suggestion
import { CommentsList } from './CommentsList/CommetsList';
```
файлы называются камелкейсом, как компонент